NewForce-Cohort-9/newforce-c9-tabloid-full-stack-TabloidFullStack
# Ticket #15: Delete a Tag

## Description
When an admin is on the Tag Management page they should see a list of all tags listed alphabetically. [Trello Ticket #15](https://trello.com/c/tEbUxGQT/15-15-delete-a-tag)

## Type of change

## Testing Instructions
1. `git add` and `git commit` whatever you're currently working on
2. `git checkout main`
3. `git fetch --all`
4. `git checkout bp-edit`
5. If you don't see the most recent version of my files, `git pull origin bp-edit`
6. If you haven't created the Tabloid database, please do so.
7. Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-api-all-stars/blob/main/SQL/01_Tabloid_Create_DB.sql)
8. Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-api-all-stars/blob/main/SQL/01_Tabloid_Create_DB.sql)
9. Click the execute with debugger solid green button in purple Visual Studio to launch the api
10. In your terminal, launch the program within the root folder with the command `npm start`

### Then:
* In the browser, please go to Login on the nav bar and log in with this email: `foo@bar.com`
* This should redirect you back to the home page
* Select `Tag Management` to be redirected to `/Tags` to see a list of all tags ordered alphabetically
* From there, click `Create Tag` to be redirected to `/newTag` where the form for adding a new tag will appear
* Enter `Test` under `Tag Name`
* Click `Save`
* After clicking `Save`, you will be redirected back to `/Tags` where your saved tag will appear in alphabetical order in the list
* After this, click `Delete`
* This will redirect you to `/deleteTag`
* Click `Deny` to be redirected back to the `Tag Management` home page
* Click `Confirm` to be redirected to a form asking if you want to delete your selected tag
* After confirming your choice, you will be redirected back to `Tag Management` home page
* From there, you will see that your created tag is no longer listed

## Checklist:
